FSharp.sln

```diff
@@ -31,7 +31,7 @@ Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "FSharp.Tests.FSharpSuite",
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "FSharp.Compiler.UnitTests", "tests\FSharp.Compiler.UnitTests\FSharp.Compiler.UnitTests.fsproj", "{A8D9641A-9170-4CF4-8FE0-6DB8C134E1B5}"
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "FSharp.Core.UnitTests", "tests\FSharp.Core.UnitTests\FSharp.Core.UnitTests.fsproj", "{88E2D422-6852-46E3-A740-83E391DC7973}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "FSharp.Core.UnitTests", "tests\FSharp.Core.UnitTests\FSharp.Core.Unittests.fsproj", "{88E2D422-6852-46E3-A740-83E391DC7973}"
```

Comment:
Good grief!!!! we should probably fix this casing by renaming the file on disk, rather than in the projects.  No one is ever going to remember to lowercase the tests.

Reply:
Yes, this one was missed when I normalized the file names.

However I've had hell sometimes with git when changing casing of filenames, so for the moment lets leave this. We can fix it later

Reply:
Yes. It's hard to change casing on windows. It's nearly impossible to get it spread to all devs and not regress by someone.